### PR TITLE
feat: vue version upgrade and sso

### PR DESCRIPTION
**Pull request by Gaurabag**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by allendevops** [comment]
### Git leaks report

Description | File | Line | Secret
-------------|------|------|--------
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | browser/flagr-ui/src/utils/apiUtil.js | 117-117 | `ey****DE`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 17-17 | `ey****Nw`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 26-26 | `ey****PM`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 29-29 | `ey****TM`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 32-32 | `ey****Aw`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 339-340 | `"p****d"`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 357-358 | `"p****d"`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 380-381 | `"p****d"`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/handler/subject_test.go | 63-63 | `ey****EI`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/handler/subject_test.go | 74-74 | `ey****y4`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/alice.key | 1-51 | `--****--`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/bob.key | 1-51 | `--****--`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/carol.key | 1-51 | `--****--`

**Comment by allendevops** [comment]
### Git leaks report

Description | File | Line | Secret
-------------|------|------|--------
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/handler/subject_test.go | 63-63 | `ey****EI`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/handler/subject_test.go | 74-74 | `ey****y4`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/alice.key | 1-51 | `--****--`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/bob.key | 1-51 | `--****--`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 17-17 | `ey****Nw`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 26-26 | `ey****PM`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 29-29 | `ey****TM`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 32-32 | `ey****Aw`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 339-340 | `"p****d"`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 357-358 | `"p****d"`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 380-381 | `"p****d"`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/carol.key | 1-51 | `--****--`

**Comment by allendevops** [comment]
### Git leaks report

Description | File | Line | Secret
-------------|------|------|--------
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/handler/subject_test.go | 63-63 | `ey****EI`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/handler/subject_test.go | 74-74 | `ey****y4`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/alice.key | 1-51 | `--****--`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/bob.key | 1-51 | `--****--`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/carol.key | 1-51 | `--****--`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 17-17 | `ey****Nw`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 26-26 | `ey****PM`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 29-29 | `ey****TM`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 32-32 | `ey****Aw`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 339-340 | `"p****d"`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 357-358 | `"p****d"`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 380-381 | `"p****d"`

**Comment by allendevops** [comment]
### Git leaks report

Description | File | Line | Secret
-------------|------|------|--------
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/handler/subject_test.go | 63-63 | `ey****EI`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/handler/subject_test.go | 74-74 | `ey****y4`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/alice.key | 1-51 | `--****--`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 18-18 | `ey****Nw`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 27-27 | `ey****PM`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 30-30 | `ey****TM`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/bob.key | 1-51 | `--****--`
Uncovered a JSON Web Token, which may lead to unauthorized access to web applications and sensitive user data. | pkg/config/middleware_test.go | 33-33 | `ey****Aw`
Identified a Private Key, which may compromise cryptographic security and sensitive data encryption. | pkg/handler/testdata/certificates/carol.key | 1-51 | `--****--`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 340-341 | `"p****d"`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 358-359 | `"p****d"`
Identified a HashiCorp Terraform password field, risking unauthorized infrastructure configuration and security breaches. | pkg/config/middleware_test.go | 381-382 | `"p****d"`

**Comment by allendevops** [comment]
### Sonar scan report

:x: Sonar quality gate failed, open [sonar dashboard](https://sonarqube.allen-live.in/dashboard?id=flagr&pullRequest=3) for more details.

Exclusions: `,*.html,*.xml,*.yml,*.yaml,*.json,*.out,*.py,**/mock*.go,**/_gen.go`

### Sonar detailed coverage report

New code coverage is: 22.4%

File name | coverage
--------- | --------
[configure_flagr.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/configure_flagr.go) | 0.0%
[constraint.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/entity/constraint.go) | 97.0%
[create_constraint_parameters.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/constraint/create_constraint_parameters.go) | 0.0%
[create_constraint_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/constraint/create_constraint_responses.go) | 0.0%
[create_flag_parameters.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/flag/create_flag_parameters.go) | 0.0%
[create_flag_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/flag/create_flag_responses.go) | 0.0%
[create_segment_parameters.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/segment/create_segment_parameters.go) | 0.0%
[create_segment_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/segment/create_segment_responses.go) | 0.0%
[create_tag_parameters.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/tag/create_tag_parameters.go) | 0.0%
[create_tag_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/tag/create_tag_responses.go) | 0.0%
[create_variant_parameters.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/variant/create_variant_parameters.go) | 0.0%
[create_variant_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/variant/create_variant_responses.go) | 0.0%
[crud.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/crud.go) | 93.1%
[crud_flag_creation.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/crud_flag_creation.go) | 86.0%
[data_record_frame.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/data_record_frame.go) | 93.8%
[data_recorder.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/data_recorder.go) | 100.0%
[data_recorder_kafka.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/data_recorder_kafka.go) | 80.2%
[data_recorder_kinesis.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/data_recorder_kinesis.go) | 86.8%
[data_recorder_pubsub.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/data_recorder_pubsub.go) | 56.3%
[db.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/entity/db.go) | 77.5%
[delete_constraint_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/constraint/delete_constraint_responses.go) | 0.0%
[delete_flag_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/flag/delete_flag_responses.go) | 0.0%
[delete_segment_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/segment/delete_segment_responses.go) | 0.0%
[delete_tag_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/tag/delete_tag_responses.go) | 0.0%
[delete_variant_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/variant/delete_variant_responses.go) | 0.0%
[e2r.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/mapper/entity_restapi/e2r/e2r.go) | 0.0%
[error.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/error.go) | 100.0%
[eval.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/eval.go) | 87.2%
[eval_cache.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/eval_cache.go) | 87.0%
[eval_cache_fetcher.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/eval_cache_fetcher.go) | 91.0%
[export.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/export.go) | 81.1%
[find_all_tags_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/tag/find_all_tags_responses.go) | 0.0%
[find_constraints_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/constraint/find_constraints_responses.go) | 0.0%
[find_distributions_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/distribution/find_distributions_responses.go) | 0.0%
[find_flags_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/flag/find_flags_responses.go) | 0.0%
[find_segments_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/segment/find_segments_responses.go) | 0.0%
[find_tags_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/tag/find_tags_responses.go) | 0.0%
[find_variants_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/variant/find_variants_responses.go) | 0.0%
[fixture.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/fixture.go) | 100.0%
[fixture.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/entity/fixture.go) | 100.0%
[flag.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/entity/flag.go) | 96.9%
[flag_snapshot.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/entity/flag_snapshot.go) | 43.8%
[flagr_api.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/flagr_api.go) | 0.0%
[get_export_eval_cache_json_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/export/get_export_eval_cache_json_responses.go) | 0.0%
[get_export_sqlite_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/export/get_export_sqlite_responses.go) | 0.0%
[get_flag_entity_types_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/flag/get_flag_entity_types_responses.go) | 0.0%
[get_flag_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/flag/get_flag_responses.go) | 0.0%
[get_flag_snapshots_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/flag/get_flag_snapshots_responses.go) | 0.0%
[get_health_responses.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/restapi/operations/health/get_health_responses.go) | 0.0%
[handler.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/handler/handler.go) | 89.3%
[jwt_middleware.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:pkg/config/jwtmiddleware/jwt_middleware.go) | 73.6%
[main.go](https://sonarqube.allen-live.in/component_measures?id=flagr&metric=new_lines_to_cover&pullRequest=3&selected=flagr:swagger_gen/cmd/flagr-server/main.go) | 0.0%


### Sonar detailed issues report

File | Line | Message
----------- | --------- | --------
[pkg/config/middleware.go](https://sonarqube.allen-live.in/project/issues?resolved=false&pullRequest=3&id=flagr&open=AZKazQiRwqygpF4gP3C8) | 40 | Refactor this method to reduce its Cognitive Complexity from 20 to the 15 allowed.
[Dockerfile](https://sonarqube.allen-live.in/project/issues?resolved=false&pullRequest=3&id=flagr&open=AZKazQiswqygpF4gP3C9) | 4 | Replace `as` with upper case format `AS`.
[browser/flagr-ui/src/components/Callback.vue](https://sonarqube.allen-live.in/project/issues?resolved=false&pullRequest=3&id=flagr&open=AZKazQeawqygpF4gP3C1) | 16 | ../utils/apiUtil import is duplicated.
[browser/flagr-ui/src/components/Callback.vue](https://sonarqube.allen-live.in/project/issues?resolved=false&pullRequest=3&id=flagr&open=AZKazQeawqygpF4gP3C2) | 18 | ../utils/apiUtil import is duplicated.
[browser/flagr-ui/src/components/Callback.vue](https://sonarqube.allen-live.in/project/issues?resolved=false&pullRequest=3&id=flagr&open=AZKazQeawqygpF4gP3C3) | 28 | Remove this commented out code.
[browser/flagr-ui/src/components/DebugConsole.vue](https://sonarqube.allen-live.in/project/issues?resolved=false&pullRequest=3&id=flagr&open=AZKazQeYwqygpF4gP3C0) | 155 | Remove this commented out code.
[browser/flagr-ui/src/components/Flags.vue](https://sonarqube.allen-live.in/project/issues?resolved=false&pullRequest=3&id=flagr&open=AZKazQd9wqygpF4gP3Cz) | 159 | @element-plus/icons import is duplicated.
[browser/flagr-ui/src/components/Login.vue](https://sonarqube.allen-live.in/project/issues?resolved=false&pullRequest=3&id=flagr&open=AZKazQefwqygpF4gP3C4) | 14 | ../utils/apiUtil import is duplicated.
[browser/flagr-ui/src/components/Login.vue](https://sonarqube.allen-live.in/project/issues?resolved=false&pullRequest=3&id=flagr&open=AZKazQefwqygpF4gP3C5) | 20 | Consider removing this try statement as promise rejection is already captured by .catch() method.
[browser/flagr-ui/src/utils/apiUtil.js](https://sonarqube.allen-live.in/project/issues?resolved=false&pullRequest=3&id=flagr&open=AZKazQejwqygpF4gP3C7) | 135 | Refactor this code to not use nested template literals.

File: browser/flagr-ui/src/utils/apiUtil.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+		throw new Error("Axios instance not initialized")
+	}
+	const token = localStorage.getItem('tokens');
+
+	// Try to parse the token and catch any potential errors
+	if(token){
+		try {
+			const authToken = token ? JSON.parse(token) : null;
+			if (authToken && authToken["x-access-token"]) {
+				axiosInstance.defaults.headers["Authorization"] = "Bearer " + authToken["x-access-token"];
```
Comment (sanket-allen) [inline browser/flagr-ui/src/utils/apiUtil.js:92]: move the string to constants

File: pkg/config/jwtmiddleware/jwt_middleware.go [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+		}
+
+		h.ServeHTTP(w, r)
+	})
+}
+
+// FromAuthHeader is a "TokenExtractor" that takes a give request and extracts
+// the JWT token from the Authorization header.
+func FromAuthHeader(r *http.Request) (string, error) {
+	authHeader := r.Header.Get("Authorization")
```
Comment (sanket-allen) [inline pkg/config/jwtmiddleware/jwt_middleware.go:128]: move to constants

File: pkg/config/jwtmiddleware/jwt_middleware.go [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+	}
+
+	return authHeaderParts[1], nil
+}
+
+// FromParameter returns a function that extracts the token from the specified
+// query string parameter
+func FromParameter(param string) TokenExtractor {
+	return func(r *http.Request) (string, error) {
+		return r.URL.Query().Get(param), nil
```
Comment (sanket-allen) [inline pkg/config/jwtmiddleware/jwt_middleware.go:146]: do we need nil checks here ?

**Review by sanket-allen — DISMISSED** [review]

File: pkg/config/jwtmiddleware/jwt_middleware.go (hunk 3 above)
Comment (himanshu-allen) [reply pkg/config/jwtmiddleware/jwt_middleware.go:154]: added

File: browser/flagr-ui/src/utils/apiUtil.js (hunk 1 above)
Comment (himanshu-allen) [reply browser/flagr-ui/src/utils/apiUtil.js:92]: this is from js file, will inform gaurab to fix this

**Review by sanket-allen — APPROVED** [review]

**Pull request merged**